lsp.el

```diff
@@ -815,9 +836,9 @@ WORKSPACE is the workspace that contains the diagnostics."
         (run-hooks 'lsp-after-diagnostics-hook)))))
 
 (with-eval-after-load 'flymake
-  (put 'lsp-note 'flymake-category 'flymake-note)
-  (put 'lsp-warning 'flymake-category 'flymake-warning)
-  (put 'lsp-error 'flymake-category 'flymake-error)
+  ;; (put 'lsp-note 'flymake-category 'flymake-note)
```

Comment:
https://github.com/emacs-mirror/emacs/blob/master/lisp/progmodes/flymake.el#L81

A different symbol `lsp-note` gives more flexibility.

Reply:
It works here.

Reply:
Ok, I will recompile latest emacs and I will test on emacs 26.1